lib/util/support.js

```diff
@@ -24,24 +24,12 @@ function isGetUserMediaSupported() {
     || !!(navigator.mozGetUserMedia);
 }
 
-/**
- * @returns {boolean} - true if the browser is explicitly unsupported.
- *  Note: this function returning false does not mean that browser is supported.
- */
-function isExplicitlyUnsupportedBrowser() {
-  if (typeof navigator !== 'undefined' && typeof navigator.userAgent === 'string') {
-    return /Edg(e)?/.test(navigator.userAgent);
-  }
-  return false;
-}
-
 /**
  * Check if the current environment is supported by the SDK.
  * @returns {boolean}
  */
 function isSupported() {
-  return !isExplicitlyUnsupportedBrowser()
```

Comment:
We have 2 flavors of Edge (Chromium & Non-Chromium). Would this return `false` for non chromium based browser?

Reply:
@syerrapragada Isn't the other browser decommissioned? Also, I think the old browser would return true because WebRTC is not supported. We can test this as part of QA.